### PR TITLE
Fix480: 	Add description for Gallery Paragraph fields

### PR DESCRIPTION
**Pull request by istankevych**

### CODE REVIEW

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1470 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1470 |
| Installation process | http://install.openy-dev.ffwua.com/build1470/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1470/

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1474 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1474 |
| Installation process | http://install.openy-dev.ffwua.com/build1474/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1474/

File: themes/openy_themes/openy_rose/templates/node/node--blog--teaser.html.twig [hunk 1]
```diff
@@ -81,26 +81,26 @@
 ] %}
 {% set category_default = node.bundle|clean_class|capitalize %}
 {% set category = content.field_blog_category.0 ? content.field_blog_category.0 : category_default|t %}
-{% set color = 'color: #' ~ content.field_blog_text_color['#items'].entity.field_color.value %}
-{% set bg_color = 'background-color: #' ~ content.field_blog_color['#items'].entity.field_color.value %}
+{% set color = content.field_blog_text_color['#items'].entity.field_color.value ? 'color: #' ~ content.field_blog_text_color['#items'].entity.field_color.value ~ ';' : null %}
```
Comment (hamrant) [inline themes/openy_themes/openy_rose/templates/node/node--blog--teaser.html.twig:84]: All changes in themes/openy_themes/openy_rose/templates/node/node--blog--teaser.html.twig not related to current issue, please remove them from PR

**Review by hamrant — CHANGES_REQUESTED** [review]

**Comment by alexschedrov** [comment]
This has been merged here #575.

**Pull request closed without merging**